internal/controller/pkg/runtime/runtime_function.go

```diff
@@ -182,14 +182,36 @@ func (h *FunctionHooks) Deactivate(ctx context.Context, _ v1.PackageRevisionWith
 	return nil
 }
 
-func functionDeploymentOverrides(image string) []DeploymentOverride {
+func functionDeploymentOverrides(pr v1.PackageRevisionWithRuntime, image string) []DeploymentOverride {
 	do := []DeploymentOverride{
 		DeploymentRuntimeWithAdditionalPorts([]corev1.ContainerPort{
 			{
 				Name:          GRPCPortName,
 				ContainerPort: GRPCPort,
 			},
 		}),
+		DeploymentRuntimeWithAdditionalEnvironments([]corev1.EnvVar{
+			{
+				Name: "FUNCTION_NAME",
+				ValueFrom: &corev1.EnvVarSource{
+					FieldRef: &corev1.ObjectFieldSelector{
+						FieldPath: "metadata.labels['pkg.crossplane.io/function']",
```

Comment:
any desire to make these labels a common const? i think they originally get created here: https://github.com/crossplane/crossplane/blob/main/internal/controller/pkg/runtime/runtime.go#L334-L335. I don't feel strongly myself, but wanted to see how you felt.